make/command

```diff
@@ -33,7 +33,7 @@ bin/print$(EXE) bin/stansummary$(EXE) bin/diagnose$(EXE) : bin/%$(EXE) : bin/cmd
 $(BOOST_PROGRAM_OPTIONS_LIB):
 	@mkdir -p $(dir $@)
 ifeq ($(OS),Windows_NT)
-	cd $(BOOST); sh bootstrap.sh
+	cd $(BOOST); ./bootstrap.bat gcc
```

Comment:
For Windows we must use the .bat script and set gcc as it otherwise tries to use msvc if it finds it.